src/components/map-page-components/map-view/MapView.tsx

```diff
@@ -1,7 +1,203 @@
+      map.setZoom(16);
+    }
+  });
+
+  const privateStreetsLayer = new google.maps.Data({ map });
+
+  privateStreetsLayer.loadGeoJson(
+    'https://raw.githubusercontent.com/florisdobber/SFTT-map-test/master/private_streets.json',
+  );
+
+  function setPrivateStreetsStyle(v: boolean) {
+    privateStreetsLayer.setStyle({
+      strokeColor: 'red',
+      strokeWeight: 2,
+      visible: v,
+    });
+  }
+
+  setPrivateStreetsStyle(false);
+
+  const blocksLayer = new google.maps.Data({ map });
+  blocksLayer.loadGeoJson(
+    'https://raw.githubusercontent.com/florisdobber/SFTT-map-test/master/blocks.json',
+  );
+
+  function setBlocksStyle(v: boolean) {
+    blocksLayer.setStyle((feature) => {
+      let color = 'green';
+
+      // Use this for coloring reserved/completed blocks a different color
+      if (feature.getProperty('ID') % 10 === 0) {
+        color = 'yellow';
+      }
+
+      if (feature.getProperty('ID') % 10 === 1) {
+        color = 'red';
+      }
+
+      // Use this for selecting blocks
+      if (feature.getProperty('PRECINCT') === 'YES') {
+        color = 'black';
+      }
+
+      // Set styling here
+      return {
+        fillColor: color,
+        strokeColor: 'black',
+        strokeWeight: 1,
+        visible: v,
+      };
+    });
+  }
+
+  setBlocksStyle(false);
+
+  const neighborhoodsLayer = new google.maps.Data({ map });
+  neighborhoodsLayer.loadGeoJson(
+    'https://raw.githubusercontent.com/florisdobber/SFTT-map-test/master/neighborhoods_edited.geojson',
+  );
+
+  function setNeighborhoodsStyle(v: boolean) {
+    neighborhoodsLayer.setStyle((feature) => {
+      return {
+        fillColor: 'green',
+        fillOpacity: (feature.getProperty('Neighborhood_ID') / 100) * 2 + 0.1, // replace this with completion percentage
+        strokeWeight: 1,
+        strokeColor: 'white',
+        visible: v,
+      };
+    });
+  }
+
+  setNeighborhoodsStyle(true);
+
+  // Check for clicks on neighborhoods and zoom to when clicked on a neighborhood
+  neighborhoodsLayer.addListener('click', (event) => {
+    map.setZoom(14);
+    map.panTo({
+      lat: event.feature.getProperty('Latitude'),
+      lng: event.feature.getProperty('Longitude'),
+    });
+  });
+
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(
+      (pos) => {
+        const me = new google.maps.LatLng(
+          pos.coords.latitude,
+          pos.coords.longitude,
+        );
+        // eslint-disable-next-line
+        const userLocation = new google.maps.Marker({
+          position: me,
+          map,
+          clickable: false,
+          icon: {
+            path: google.maps.SymbolPath.CIRCLE,
+            scale: 5,
+            fillColor: 'blue',
+            fillOpacity: 1,
+            strokeWeight: 0,
+          },
+        });
+      },
+      (error) => {
+        // Handle the error
+        // tslint:disable-next-line
+        console.log('Location not found');
```

Comment:
```suggestion
        // TODO: Handle the error
```
How does this error affect UX? what should we do to notify the user properly, if needed? Console.log doesn't give us much.

Reply:
Forgot to delete console.log, was for testing. New descriptive todo tag showing what needs to be done.